### PR TITLE
Ll insertions

### PR DESCRIPTION
**Pull request by egconley**
- [x] Top-level README “Table of Contents” is updated
 - [X] Feature tasks for this challenge are completed
 - [X] Unit tests written and passing
     - [X] “Happy Path” - Expected outcome
     - [ ] Expected failure
     - [ ] Edge Case (if applicable/obvious)
 - [x] README for this challenge is complete
     - [x] Summary, Description, Approach & Efficiency, Solution
     - [x] Link to code
     - [x] Picture of whiteboard

### CODE REVIEW

File: code401challenges/src/main/java/code401challenges/linkedlist/LinkedList.java [hunk 1]
```diff
@@ -69,19 +125,20 @@ private Node find(String target) {
     // Define a method called toString (or __str__ in Python) which takes in no arguments
     // and returns a string representing all the values in the Linked List, formatted as:
     // "{ a } -> { b } -> { c } -> NULL"
+
+    // update this to match with new toString() method
     public String toString() {
         Node position = head;
         String LinkedListString = "";
```
Comment (Zaffierce) [inline code401challenges/src/main/java/code401challenges/linkedlist/LinkedList.java:132]: Instead of defining your LinkedListString as a String, you could do a StringBuilder.
`StringBuilder LinkedListString = new StringBuilder();`

File: code401challenges/src/main/java/code401challenges/linkedlist/LinkedList.java [hunk 2]
```diff
@@ -69,19 +125,20 @@ private Node find(String target) {
...
         while (position != null) {
             //String itemAsString = "{ " + position.getItem() + ", " + position.getCount() + " }";
+            //could use .append (string builder)
             String itemAsString = "{ " + position.getItem() + " }";
             LinkedListString += itemAsString + " -> ";
```
Comment (Zaffierce) [inline code401challenges/src/main/java/code401challenges/linkedlist/LinkedList.java:137]: using StringBuilder, you could then append.
`LinkedListString.append("{ " + position.getItem() + " } -> ");`

File: code401challenges/src/main/java/code401challenges/linkedlist/LinkedList.java [hunk 3]
```diff
@@ -69,19 +125,20 @@ private Node find(String target) {
...
 
             position = position.getLink();
         }
-        if (position == null) {
-            LinkedListString += "NULL";
-        }
+        LinkedListString += "NULL";
```
Comment (Zaffierce) [inline code401challenges/src/main/java/code401challenges/linkedlist/LinkedList.java:141]: and here, now you can do one last append for NULL.
`LinkedListString.append("NULL");`

File: code401challenges/src/main/java/code401challenges/linkedlist/LinkedList.java [hunk 4]
```diff
@@ -69,19 +125,20 @@ private Node find(String target) {
...
         return LinkedListString;
```
Comment (Zaffierce) [inline code401challenges/src/main/java/code401challenges/linkedlist/LinkedList.java:142]: Finally, and something that may seem silly... You have to add a `.toString()` on the end of your LinkedListString.
